packages/demo/src/components/examples/ToastConnectedExamples.tsx

```diff
@@ -94,7 +95,21 @@ export class ToastConnectedExamples extends React.Component<IToastConnectedExamp
                             React Component Content with children
                         </button>
 
+                        <button
```

Comment:
What do you think of using our own Button component instead?

Reply:
I think its a good idea but I'd like to hold off changing too much about he demo until all the new Rebranding changes are in